arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -24,18 +24,10 @@
         "operationId": "AlertRuleIncidents_Get",
```

Comment:
```
"info": {
     "title": "InsightsManagementClient",
     "version": "2016-03-01"
   },
```
the version value in the above object and the one in the path to this file `arm-insights/2015-04-01/swagger/insightsManagementClient.json` should be the same.